Documentation/Books/Manual/Architecture/DeploymentModes/ActiveFailover/Limitations.md

```diff
@@ -4,8 +4,11 @@ Active Failover Limitations
 The _Active Failover_ setup in ArangoDB has a few limitations. Some of these limitations 
 may be removed in later versions of ArangoDB:
 
-- Even though it is already possible to have several _followers_ of the same _leader_,
- currently only one _follower_ is officially supported
 - Should you add more than one _follower_, be aware that during a _failover_ situation
- the failover attempts to pick the most up to date follower as a new leader, 
- but there is **no guarantee** on how much operations may have been lost.
+ the failover attempts to pick the most up to date follower as the new leader on a **best-effort** basis. 
+- In contrast to full ArangoDB Cluster (with synchronous replication), there is **no guarantee** on 
+  how many database operations may have been lost during a failover.
+- Should you be using the [ArangoDB Starter](https://github.com/arangodb-helper/arangodb) 
+  or the [Kubernetes Operator](https://github.com/arangodb/kube-arangodb) to manage your Active-Failover
```

Comment:
something like ../../../Deployment/Kubernetes/README.md (same reason as before - no need to point users to external repos)